packages/web/config/ibc-assets.ts

```diff
@@ -717,6 +717,15 @@ export const IBCAssetInfos: (IBCAsset & {
     destChannelId: "channel-3",
     coinMinimalDenom: "ukuji",
   },
+  {
+    counterpartyChainId: "echelon_3000-3",
+    sourceChannelId: "channel-262",
+    destChannelId: "channel-8",
+    coinMinimalDenom: "aechelon",
+    depositUrlOverride: "https://app.ech.network/ibc",
+    withdrawUrlOverride: "https://app.ech.network/ibc",
+    isVerified: true,
```

Comment:
```suggestion
```

Reply:
We will not be adding this asset to the main frontend rn.  Only the permissionless frontier.

Reply:
Ack and updated! Thank you @sunnya97

Reply:
Conventionally, we only include assets on the main frontend that are receiving incentives. 

Though, with current market conditions we are proposing to revisit our incentive strategy by redirecting some to the community pool. @sunnya97 thoughts?